app/src/main/java/com/amaze/filemanager/activities/MainActivity.java

```diff
@@ -1329,12 +1333,12 @@ protected void onDestroy() {
     // TODO: https://developer.android.com/reference/android/app/Activity.html#onDestroy%28%29
     closeInteractiveShell();
 
-    tabHandler.close();
-    utilsHandler.close();
-    cloudHandler.close();
-
-    CryptHandler cryptHandler = new CryptHandler(this);
-    cryptHandler.close();
+    //        if(ExplorerDatabase.getInstance() != null && ExplorerDatabase.getInstance().isOpen())
```

Comment:
Can remove commented code if not needed? BTW not closing the connection on exit?